### ISSUE TITLE
Tooling?

### ISSUE THREAD

**Issue by clbarnes** [bibtxt/bibtxt.github.io#1]
The existence of another way to serialise bibliographies is... fine, I guess. But it's not worth making a machine-readable format without providing a way for machines to read it! Is there any tooling available? Is there a specification? Is it a subset of another file format (e.g. ini)? Why does this format exist?

**Comment by geraldb** [comment]
Good question Why do you exist? Why does Mount Everest exist? Why do humans exist?

**Comment by clbarnes** [comment]
I apologise if I came off as insulting, I certainly didn't mean to. I was posing questions which I would expect any potential user to have, and which I couldn't glean answers to from the existing documentation. I'm very much in favour of better, more portable ways to share citation data and am excited by the projects built for that purpose. However, if the only available documentation is "I've made up a file format and I'm not telling you any more than that", it just adds noise to the ecosystem, rather than any benefit.

Of course, if bibTXT is a naturally-occurring phenomenon, that's also an acceptable (if surprising) thing to advertise!
